api/envoy/config/grpc_credential/v2alpha/aws_iam.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+// [#protodoc-title: Grpc Credentials AWS IAM]
```

Comment:
Please add this to the docs build (see commits to https://github.com/envoyproxy/envoy/blob/master/docs/build.sh for how to do this)

Reply:
done